src/Tribe/Repositories/Event.php

```diff
@@ -1653,10 +1670,14 @@ protected function order_by_organizer( $after = false ) {
 	 * @since 4.9.7
 	 * @since 4.9.11 Added the `$after` parameter.
 	 *
-	 * @param bool $after        Whether to append the order by clause to the ones managed by WordPress or not.
+	 * @param string $order      The order direction, either `ASC` or `DESC`; defaults to `null` to use the order
+	 *                           specified in the current query or default arguments.
+	 * @param bool   $after      Whether to append the order by clause to the ones managed by WordPress or not.
 	 *                           Defaults to `false`,to prepend them to the ones managed by WordPress.
+	 * @param bool   $override   Whether to override existing ORDER BY clauses with this one or not; default to
+	 *                           `true` to override existing ORDER BY clauses.
 	 */
-	protected function order_by_venue( $after = false ) {
+	protected function order_by_venue( $order = null,$after = false, $override = true ) {
```

Comment:
* :no_entry_sign: Expected 1 space between comma and argument "$after"; 0 found
* :no_entry_sign: Expected 1 space between comma and argument "$after"; 0 found

<sub>hash: b76879ee15fe268d36d0188694f0c036</sub>